### ISSUE TITLE
[ui-upgrade]: 

### ISSUE THREAD

**Issue by sahashorha** [SyedImtiyaz-1/PreciousServices#175]
**Describe the bug**
![Screenshot (251)](https://github.com/SyedImtiyaz-1/PreciousServices/assets/102842190/2e3546d3-bb72-4997-9f5e-d62e4f5ec2d2)

In this issue Add Css to the footer for better experience and user interface
🌟 **Note**: Please make sure to complete all the required tasks before submitting your issue/PR.

- [ ] I have checked the existing issues
- [ ] [Give a Star](https://github.com/SyedImtiyaz-1/PreciousServices) ⭐
- [ ] Follow - [Project Admin](https://github.com/SyedImtiyaz-1/)
- [ ] I want to work on this issue

Connect with me on [Linkedin](https://linkedin.com/in/imtiyaz-sde)
Let's Connect 1:1 - [Click Here]https://topmate.io/syedimtiyazali

**Comment by sahashorha** [comment]
Can u assign me this task

**Comment by kabir-afk** [comment]
> ![Screenshot (252)](https://user-images.githubusercontent.com/102842190/272010862-a39cc7a2-6833-4a8b-bafc-54b5191564e2.png)
> 
> when i add the css this error shown what i do to overcome this error please help me

can we work together on this one ? I might be able to help

**Comment by sahashorha** [comment]
YES PLEASE IF U HELP ME THAN I WILL ASSURE U I WILL MAKE THIS PROJECT AT ANOTHER LEVEL BRO

**Comment by kabir-afk** [comment]
well, I reviewed and it is a backend issue which includes parsing the MIME type from 'text/html' to css or csv , can't really help with that brdr . . .consult someone else whos good in backend
